### PR TITLE
Featute/add green buttons

### PR DESCRIPTION
**Pull request by barakpazlab**
added success green colors (incl. dark and light green); added success buttons (+ outlined and disabled); added color codes to all color on color Palette. 
build process wants to change the font files as well but we don't know if we should commit them or not
please advise.

### CODE REVIEW

**Comment by barakpazlab** [comment]
added the code examples for big green buttons and also added small green buttons (normal/outlined/disabled)

File: src/scss/_buttons.scss [hunk 1]
```diff
@@ -98,6 +98,36 @@
     }
   }
 
+     &.btn-success {
+    background-color: $color-success;
```
Comment (algo26-matthias) [inline src/scss/_buttons.scss]:
```suggestion
         background-color: $color-success;
```

File: src/scss/_buttons.scss [hunk 2]
```diff
@@ -119,6 +149,29 @@
       color: $color-dark;
     }
   }
+    
+      &.btn-success-outline {
+    background-color: $color-white;
```
Comment (algo26-matthias) [inline src/scss/_buttons.scss]:
```suggestion
          background-color: $color-white;
```

Comment (algo26-matthias) [reply src/scss/_buttons.scss]: Please indent properly

File: src/scss/_buttons.scss (hunk 1 above)
Comment (algo26-matthias) [reply src/scss/_buttons.scss]: Also here: Indentation.

**Comment by bgraf-zlab** [comment]
@algo26-matthias @barakpazlab we accidently checked in those font files, that gulp or webpack generated automatically… Do you know what those are about? Should we revert those changes? Or do you know who knows more about those?

**Comment by algo26-matthias** [comment]
Unless you changed anything in the icons you can safely revert the changes to the font files. These are always generated whenever you rebuild the project.
One thing I missed: Please increase the version of the project in the `packaga.json`. Also please don't forget to tag the commit accordingly once it's been merged to master.

**Comment by bgraf-zlab** [comment]
Allright, I just reverted the font files.

> One thing I missed: Please increase the version of the project in the `packaga.json`. Also please don't forget to tag the commit accordingly once it's been merged to master.

 - Do you use semantic versioning, so go to `1.3.2`? 
 - Do I understand you correctly in that we need to tag _after_ merging to master, thus the 'merge commit' that's created by merging?

**Comment by kayes** [comment]
@bgraf-zlab `1.3.2` is right for `package.json`. After it's got merged you can tag it with `v1.3.2`.

**Pull request merged**